practice_06.08.2019_System.IO/Task3_FileCharacters/FileCharacters.cs

```diff
@@ -0,0 +1,65 @@
+﻿using practice_06._08._2019_System.IO.Task2_FileCreation;
+using System;
+using System.IO;
+
+//Разработайте приложение, которое позволяет посчитать сколько содержится в файле символов, строк и слов.
+//Разделителями слов считать пробелы и перенос строки.
+
+namespace practice_06._08._2019_System.IO.Task3_FileCharacters
+{
+    public class FileCharacters
+    {
+        public void CountCharacters(string filePath)
+        {
+            string tempText = "";
+            char[] charArray;
+            string[] textArray;
+            int wordCounter = 0;
+            int lineCounter = 0;
+            var file = new FileInfo(filePath);
+            StreamReader reader = new StreamReader(file.FullName);
+            tempText = reader.ReadToEnd();
+
+            Console.WriteLine($"Text in file: {Environment.NewLine}{tempText}");
+            using (reader = new StreamReader(file.FullName))
```

Comment:
Thats good.

Reply:
Yeah, thanks. But unfortunately I used the solution from the internet